mix.exs

```diff
@@ -34,6 +34,8 @@ defmodule Ecto.Mixfile do
 
   defp deps do
     [{:poolboy, "~> 1.4"},
+     {:sbroker, "~> 0.6", github: "fishcakez/sbroker", branch: "gen"},
```

Comment:
Not part of this discussion, but FWIW I'm using it on my OrientDB adapter as well :)